### DISCUSSION TITLE
fast frequentie scanner

### DISCUSSION THREAD

**Discussion by fiann21** [AlexandreRouma/SDRPlusPlus#467] [Ideas]
this 1  is superfast > maybe you can add of plugin manager 

link to frequentie scanner > https://mega.nz/folder/WOhgiYjB#_V0Fr0eu3UxFWXzytAZsIA/folder/STYE1a4T

**Comment by the-gobleo-tinkerer** [comment]
SDR# plugins cannot be used with SDR++. See #369.
